cmd/kuberhealthy/Makefile

```
@@ -1,5 +1,5 @@
 IMAGE="kuberhealthy/kuberhealthy"
-TAG="v2.3.2"
+TAG="v2.4.0"
```

```suggestion
TAG="latest"
```

Let's move this to `latest` and only tag images with release versions when we do a release.  In the future, we will just make new builds of kuberhealthy use a commit hash until a release is cut.